lib/android-helpers.js

```diff
@@ -335,6 +335,14 @@ helpers.initUnicodeKeyboard = async function (adb) {
   return defaultIME;
 };
 
+helpers.setMockLocationApp = async function (adb, app) {
+  if (await adb.getApiLevel() < 23) {
```

Comment:
I don't know if I should change that much. I can parse it into an int. There is already a call doing what I'm doing in this file. That should also change probably